docs/quickstart/executor-values.yaml

```diff
@@ -0,0 +1,8 @@
+applicationConfig:
+  apiConnection:
+    forceNoTls: true 
```

Comment:
This is unsafe and suitable only for local testing, maybe we should put comment with big warning here? If it is in file people are more likely to copy it....

Reply:
People copying insecure code for their production install is a completely reasonable assumption.... I'll add some harsh comment in the yaml and see if we feel comfortable with that.

Reply:
It's probably OK for the quickstart

Reply:
@jankaspar This work for you or should I make it more severe?